### DISCUSSION TITLE
[Android] TestID property didn't propagate to the Marker element

### DISCUSSION THREAD

**Discussion by olukash** [react-native-maps/react-native-maps#4684] [Feature requests]
### Summary

On Android the defined testID property on a Marker gets lost. It is not visible as the accessibility (or any other) identifier. On iOS it works as expected.
Looks like, when adding testID for Marker, the Android part was missed (see PR https://github.com/react-native-maps/react-native-maps/pull/2253/files)

### Reproducible sample code

```TSX
import React from 'react';
import MapView, {Marker} from 'react-native-maps';

const locations = [
  {
    latitude: 49.2988199,
    longitude: 8.6490136,
  },
  {
    latitude: 49.293936,
    longitude: 8.674228,
  },
];

const Maps = () => (
  <MapView
    style={{flex: 1}}
    initialRegion={{
      latitude: 51.58453765908476,
      latitudeDelta: 12.60450692544972,
      longitude: 10.199171528220177,
      longitudeDelta: 10.48278160393238,
    }}>
    {locations.map((l, i) => (
      <Marker key={i} coordinate={l} testID="marker" />
    ))}
  </MapView>
);

export default Maps;
```

### Steps to reproduce

1. Add testID to the Marker element
2. Use it in the sample app
3. Load this app to Android device (or use Android studio)
4. Check testID presence

NOTE: The same steps on iOS leads to testID displaying

### Expected result

testID displayed in properties

### Actual result

testID absent in properties

### React Native Maps Version

1.4.0

### What platforms are you seeing the problem on?

Android

### React Native Version

0.69.7

### What version of Expo are you using?

Not using Expo

### Device(s)

Samsung Galaxy S20

<img width="1259" alt="Знімок екрана 2023-03-17 о 16 18 34" src="https://user-images.githubusercontent.com/6576471/225952427-a8ff4125-ffb4-4fcc-b67b-4fdc939149a6.png">

**Comment by monholm** [comment]
Hi @olukash 👋

I'm currently looking into adding some detox e2e tests to rnm and stumbled upon the same issue. Would you be able to work on a PR? :)

**Comment by mkhelifi-crea** [comment]
HI guys,

Running into the same issue here: any updates?

**Comment by samtagalog** [comment]
Workaround is to wrap your Marker into View and move your testID to the View.

**Comment by borysenko-oleksandr** [comment]
Hello, I use _**description**_ as **testID**, it may help you.

**Reply by dgreasi** [reply]
This worked for me in `"react-native-maps": "^1.22.6"`
